content/docs/app-developer-guide/using-http-proxy.md

```diff
@@ -0,0 +1,54 @@
++++
+title="Using Buildpacks with a Proxy"
+weight=8
+summary="Learn how to use buildpacks behind a HTTP proxy."
++++
+
+Many university or corporate environments use a proxy to access HTTP and HTTPS resources on the web.  Proxies introduce two constraints on buildpack built applications:
+
+1. `pack` needs to download buildpacks through the proxies, and
+2. applications running in containers created by `pack` need to call APIs behind the proxy.
+
+We show how to solve both of these constraints.
+
+## Making `pack` Proxy Aware
+
+You may need the `pack` command-line tool to download buildpacks and images via your proxy.  Building an application with an incorrectly configured proxy results in errors such as the following:
+
+```console
+$ pack build sample-app --path samples/apps/java-maven --builder cnbs/sample-builder:bionic
+ERROR: failed to build: failed to fetch builder image 'index.docker.io/cnbs/sample-builder:bionic'
+: Error response from daemon: Get "https//registry-1.docker.io/v2/": context deadline exceeded
+```
+
+The `pack` tool uses the Docker daemon to manage the local image registry on your machine.  The `pack` tool will ask the Docker daemon to download buildpacks and images for you.  Because of this relationship, between `pack` and the Docker daemon, we need to configure the Docker daemon to use a HTTP proxy.
+
+The Docker project documents [how to configure configure the HTTP/HTTPS proxy](https://docs.docker.com/config/daemon/systemd/#httphttps-proxy) settings for the Docker daemon.  You should configure the `HTTP_PROXY` and `HTTPS_PROXY` environment variables as part of the Docker daemon startup.
```

Comment:
The link is linux specific which I find confusing as a Docker Desktop user. Based on [this link](https://docs.docker.com/desktop/mac/#resources), there are 2 ways to configure it for Docker Desktop. 

1. Setting [a proxy at the OS level](https://support.apple.com/guide/mac-help/enter-proxy-server-settings-on-mac-mchlp2591/mac) would be also propagate to the daemon on on Docker Desktop.
    > Docker Desktop detects HTTP/HTTPS Proxy Settings from macOS and automatically propagates these to Docker. 
2. Setting the Proxy settings in the [Resources](https://docs.docker.com/desktop/mac/#resources) section of Preferences.

Maybe it's worth denoting the difference based on the user setup. Docker Desktop and Linux.

Reply:
Thank you.  Yes, I will differentiate between Linux and Docker Desktop.

Reply:
In "Making `pack` proxy aware" I've provided Docker Desktop and Linux instructions.  I thought that was a better distinction than between Docker Desktop and Docker CE as some developers may be unfamiliar with the distinction between Desktop and CE.  I used a link to `dummies.com` for the Windows proxy configuration as I could not find an end-user focused guide in the official MS support documentation.